Modules/Core/Common/include/itkPrintHelper.h

```diff
@@ -50,6 +52,43 @@ operator<<(std::ostream & os, const std::vector<T> & v)
   return os << v.back() << ')';
 }
 
+template <typename T>
+std::ostream &
+operator<<(std::ostream & os, const std::list<T> & l)
+{
+  if (l.empty())
+  {
+    return os << "()";
+  }
+
+  os << '(';
+  auto it = l.begin();
+  auto last = std::prev(l.end());
+  for (; it != last; ++it)
+  {
+    os << *it << ", ";
+  }
+  return os << *it << ')';
```

Comment:
Preferably do the declaration of the iterator `it` within the `for`: 

```cpp
  for (auto it = l.begin(); it != last; ++it)
  {
    os << *it << ", ";
  }
  return os << *last << ')';

```

Reply:
Please also consider the implementation of the similar `operator<<(std::ostream & os, const std::vector<T> & v)` in the very same source file:

https://github.com/InsightSoftwareConsortium/ITK/blob/1cb6591fe3387d931e07e693c08292a5b199ce42/Modules/Core/Common/include/itkPrintHelper.h#L48-L50

So I would suggest using `std::copy` and `std::ostream_iterator<T>(os, ", ")` when printing `std::list` as well.